app/response_processor.py

```diff
@@ -125,7 +125,10 @@ def _send_receipt(self, decrypted, xml):
 
             try:
                 response.raise_for_status()
-                self.logger.info("Sent receipt")
+                namespace = {'receipt': 'http://ns.ons.gov.uk/namespaces/resources/receipt'}
+                tree = etree.fromstring(xml)
+                respondent_id = tree.find('receipt:respondent_id', namespace).text
+                self.logger.info("Sent receipt for ", respondent_id=respondent_id)
```

Comment:
It needs some XML exception handling here.

Though if it fails we still need to send the receipt